### PR TITLE
protocol governance spec

### PR DESCRIPTION
**Pull request by grigorevPavel**

### CODE REVIEW

**Review by MihanixA — CHANGES_REQUESTED** [review]
structure should have following describes 
`describe("access control", () => {});`
`describe("edge cases", () => {});`

File: test/ProtocolGovernance.spec.ts [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+                
+        });
+
+        describe("when there are repeating addresses", () => {
+            it("sets sets pending vault governances and pendingVaultGovernancesAddTimestamp", async () => {
+                
+            });
+        });
+
+        describe("when callen by not admin", () => {
```
Comment (MihanixA) [inline test/ProtocolGovernance.spec.ts:112]: when called by random address

File: test/ProtocolGovernance.spec.ts [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+        it("removes", async () => {
+            
+        });
+
+        describe("when call commit on removed token", () => {
+            it("passes", async () => {
+                
+            });
+        });
```
Comment (MihanixA) [inline test/ProtocolGovernance.spec.ts:305]: let's add more edge cases 
for example, attempt to remove token that hadn't been whitelisted previously

**Review by MihanixA — CHANGES_REQUESTED** [review]

File: test/ProtocolGovernance.spec.ts (hunk 2 above)
Comment (grigorevPavel) [reply test/ProtocolGovernance.spec.ts:323]: This edge case has already been implemented
